src/components/organisms/stories/GroupForm.stories.tsx

```diff
@@ -2,7 +2,7 @@ import { ComponentMeta, ComponentStory } from '@storybook/react'
 
 import { defaultGroupFormValues, GroupForm } from '../GroupForm'
 
-// TODO: need to mock wallet store for this to render
+// TODO: need to mock Wallet store for this to render
```

Comment:
I can’t test this (don’t have a linux system) but can’t imagine thsi caused issue, given it’s in a code comment?

Reply:
Yeah It seems that it is what happened

Reply:
that would be very strange - comments are removed from typescript at compilation IIRC.

Also strange is despite seeing the renamed files here on git, when I pull the branch down locally, nothing changes. I might need to have you test… just pushed a commit (after merging this) which changes all `store` files to use lowercase notation (ie `Wallet.store.ts` -> `wallet.store.ts`). 

@JeancarloBarrios can you pull down master and see if it builds for you? You’ll also need to run `yarn`, as I added some eslint + ts settings to try and avoid this happening again